### ISSUE TITLE
Alt+Tabbing briefly causes 'if no actions are performed for x seconds, switch to scene' to swap scene

### ISSUE THREAD

**Issue by LobaDK** [WarmUpTill/SceneSwitcher#542]
**Describe the bug**
Alt+Tab'ing while having Match Behavior set to 'if no actions are performed for 0.00 seconds, switch to scene' and a macro set to 'If process is running, switch to scene' causes Match behavior to briefly overrule it and swaps back and forth between them.

**To Reproduce**
1. Set Match Behavior to 'if no actions are performed for 0.00 seconds, switch to scene'
2. Create macro that swaps to another scene if the process is running
3. Alt+Tab
4. Briefly see OBS swapping back to the default scene, causing my controller overlay to reset for the GTA5 scene I have

**Expected behavior**
Nothing is expected to happen when I Alt+Tab. Older versions at least didn't do this from what I remember

**Logs**
```
00:27:09.311: [adv-ss] detected busy loop - refusing to sleep less than 1ms
00:27:09.312: [adv-ss] try to sleep for 50
00:27:09.370: [adv-ss] condition process returned 1
00:27:09.370: [adv-ss] Macro GTA5 returned 1
00:27:09.370: [adv-ss] running macro: GTA5
00:27:09.370: [adv-ss] switch to scene 'GTA5'
00:27:09.827: User switched to scene 'Every other game'
00:27:09.872: [adv-ss] detected busy loop - refusing to sleep less than 1ms
00:27:09.872: [adv-ss] try to sleep for 50
00:27:09.933: [adv-ss] condition process returned 1
00:27:09.933: [adv-ss] Macro GTA5 returned 1
00:27:09.933: [adv-ss] running macro: GTA5
00:27:09.933: [adv-ss] switch to scene 'GTA5'
00:27:09.961: User switched to scene 'GTA5'
00:27:09.961: [adv-ss] switched scene
00:27:10.346: User switched to scene 'Every other game'
00:27:10.462: [adv-ss] detected busy loop - refusing to sleep less than 1ms
00:27:10.462: [adv-ss] try to sleep for 50
00:27:10.521: [adv-ss] condition process returned 1
00:27:10.521: [adv-ss] Macro GTA5 returned 1
00:27:10.521: [adv-ss] running macro: GTA5
00:27:10.521: [adv-ss] switch to scene 'GTA5'
00:27:10.550: User switched to scene 'GTA5'
```
https://sharex.lobadk.com/QaBu2/VIxAsite54.png
https://sharex.lobadk.com/QaBu2/heMEfaHa25.png

**Version information**
 - OS: Windows 10
 - OBS Version: 28.0.1 (also happens on 28.0.2)
 - Plugin Version : 1.18.0

**Additional context**

**Comment by WarmUpTill** [comment]
I would suggest to solve this entirely via macros and not rely on the "if no actions are performed ..." option on the General tab, as this more robust and will allow you to expand upon it more easily.

Change the General tab settings to:
![DontSwitch](https://user-images.githubusercontent.com/19472752/192086553-9d70a8cf-cbd0-4820-8066-b6029724b541.PNG)

Add a second macro with the following settings:
![NotGTA](https://user-images.githubusercontent.com/19472752/192086641-3c8bce29-1764-4e49-9ce8-db124b55946d.PNG)

Make sure to select "If not" as the condition.
Hope that helps!
Let me know if you have any questions!

**Comment by LobaDK** [comment]
Unfortunately I have multiple game-specific scenes that I'd have to check, but I assume I could just chain up a bunch 'If not macro, or macro, or macro etc...' to handle that, however testing your example still causes Advanced Scene Switcher to momentarily switch to the 'Not GTA5' even while the GTA5 process is running in the background still

**Comment by LobaDK** [comment]
Before I forget, I noticed opening the start menu in Windows, either by pressing the Windows key or directly pressing the icon, also results in this. It seems that any sort of desktop interaction or window-change causes this issue.

**Comment by LobaDK** [comment]
I realized I should've just done this from the start and apologize for taking your time, but I decided to just completely delete the Advanced Scene Switcher installation, and reinstall it, which seems to have fixed it. I did attempt to revert back to OBS version 27, some days ago, including the plugins, because some older ones were incompatible and no longer updated, so I'd assume somehow downgrading either OBS itself, or the plugin, messed something up.

Thank you for your time though, and for letting me know about using 'if not' conditions instead, that does work a little better.
